app/src/main/java/org/schabi/newpipe/player/helper/PlayerDataSource.java

```diff
@@ -62,28 +56,26 @@ public PlayerDataSource(@NonNull final Context context, @NonNull final String us
     public DashMediaSource.Factory getLiveDashMediaSourceFactory() {
         return new DashMediaSource.Factory(
                 getDefaultDashChunkSourceFactory(cachelessDataSourceFactory),
-                cachelessDataSourceFactory
-        )
-                .setLoadErrorHandlingPolicy(
-                        new DefaultLoadErrorHandlingPolicy(MANIFEST_MINIMUM_RETRY));
+                cachelessDataSourceFactory);
     }
 
     private DefaultDashChunkSource.Factory getDefaultDashChunkSourceFactory(
             final DataSource.Factory dataSourceFactory
     ) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
-            return new DefaultDashChunkSource.Factory(
-                    MediaParserChunkExtractor.FACTORY,
-                    dataSourceFactory,
-                    1
-            );
+            return new DefaultDashChunkSource.Factory(MediaParserChunkExtractor.FACTORY,
+                    dataSourceFactory, 1);
         }
 
         return new DefaultDashChunkSource.Factory(dataSourceFactory);
```

Comment:
```suggestion
        } else {
            new DefaultDashChunkSource.Factory(dataSourceFactory);
        }
```

Reply:
This is probably outdated / non-applicable because of #7555. I will check after I rebase the branch when I get the conflict part.